drivers/npmx/npmx_shell.c

```diff
@@ -2209,6 +2209,150 @@ static int cmd_ldsw_soft_start_current_set(const struct shell *shell, size_t arg
 	return ldsw_soft_start_config_set(shell, argc, argv, LDSW_SOFT_START_TYPE_CURRENT);
 }
 
+static npmx_ldsw_gpio_t ldsw_gpio_convert_from_index(int gpio_idx)
+{
+	switch (gpio_idx) {
+	case (-1):
+		return NPMX_LDSW_GPIO_NC;
+	case (0):
+		return NPMX_LDSW_GPIO_0;
+	case (1):
+		return NPMX_LDSW_GPIO_1;
+	case (2):
+		return NPMX_LDSW_GPIO_2;
+	case (3):
+		return NPMX_LDSW_GPIO_3;
+	case (4):
+		return NPMX_LDSW_GPIO_4;
+	default:
+		return NPMX_LDSW_GPIO_INVALID;
+	}
+}
+
+static int cmd_ldsw_enable_gpio_get(const struct shell *shell, size_t argc, char **argv)
+{
+	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
+
+	if (npmx_instance == NULL) {
+		shell_error(shell, "Error: shell is not initialized.");
+		return 0;
+	}
+
+	if (argc < 2) {
+		shell_error(shell, "Error: missing LDSW instance index.");
+		return 0;
+	}
+
+	int err = 0;
+	uint8_t ldsw_idx = CLAMP(shell_strtoul(argv[1], 0, &err), 0, UINT8_MAX);
+
+	if (err != 0) {
+		shell_error(shell, "Error: LDSW instance must be an integer.");
+		return 0;
+	}
+
+	if (ldsw_idx >= NPM_LDSW_COUNT) {
+		shell_error(shell, "Error: LDSW instance index is too high: no such instance.");
+		return 0;
+	}
+
+	npmx_ldsw_t *ldsw_instance = npmx_ldsw_get(npmx_instance, ldsw_idx);
+
+	npmx_error_t err_code;
+	npmx_ldsw_gpio_config_t gpio_config;
+
+	err_code = npmx_ldsw_enable_gpio_get(ldsw_instance, &gpio_config);
+
+	if (!check_error_code(shell, err_code)) {
+		shell_error(shell, "Error: unable to read GPIO config.");
+		return 0;
+	}
+
+	int val = (gpio_config.gpio == NPMX_LDSW_GPIO_NC) ? -1 : ((int)gpio_config.gpio - 1);
+
+	shell_print(shell, "Value: %d %d.", val, (int)gpio_config.inverted);
```

Comment:
```suggestion
	shell_print(shell, "Value: %d %u.", val, (unsigned int)gpio_config.inverted);
```
I think it is more intuitive, but it's a minor thing, can be omitted.